README.md

```diff
@@ -1,15 +1,17 @@
 Uploadcare Widget
 =================
 
-[Uploadcare](https://uploadcare.com) is an IaaS for file uploads,
+[Uploadcare](https://uploadcare.com) is an PaaS for file uploads,
 cloud storage and on-the-fly image transformations.
 
-This package contains HTML5 widget and JavaScript API:
```

Comment:
Please submit only bower related issues to this repo, otherwise use widget's repo.